validator/client/src/main/java/tech/pegasys/teku/validator/client/ValidatorClientService.java

```diff
@@ -271,16 +271,15 @@ public static ValidatorClientService create(
             })
         .exceptionally(
             error -> {
-              ExceptionUtil.getCause(error, InvalidConfigurationException.class)
-                  .ifPresentOrElse(
-                      cause -> LOG.error(cause.getMessage()),
-                      () ->
-                          LOG.error(
-                              "Error was encountered during validator client service start up.",
-                              error));
+              LOG.error("Error was encountered during validator client service start up.", error);
```

Comment:
i dont really want this unconditionally to dump exception trace, its just noise if we know what the problem is